src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -294,6 +299,33 @@ protected function bootstrappers()
     }
 
     /**
+     * Set environment if 'env' option if provided.
+     *
+     * @param  InputInterface  $input
+     * @return void
+     */
+    protected function setEnvironment($input)
+    {
+        if (! app()->runningInConsole()) {
```

Comment:
yeh, you can. Laravel's contracts at stupid anyway. We use methods that arn't actually on them all over the place.